cmd/evm/main.go

```diff
@@ -121,17 +124,36 @@ func run(ctx *cli.Context) error {
 	})
 
 	tstart := time.Now()
-	ret, e := vmenv.Call(
-		sender,
-		receiver.Address(),
-		common.Hex2Bytes(ctx.GlobalString(InputFlag.Name)),
-		common.Big(ctx.GlobalString(GasFlag.Name)),
-		common.Big(ctx.GlobalString(PriceFlag.Name)),
-		common.Big(ctx.GlobalString(ValueFlag.Name)),
+
+	var (
+		ret []byte
+		err error
 	)
+
+	if ctx.GlobalBool(CreateFlag.Name) {
+		ret, _, err = vmenv.Create(
+			sender,
+			common.Hex2Bytes(ctx.GlobalString(CodeFlag.Name)),
```

Comment:
Please append the input here too so we might add arguments as `evm` `--input`. Otherwise LGTM